modules/http/README.md

```diff
@@ -0,0 +1,26 @@
+http destination
+================
+
+The http destination can send the log as HTTP requests to an HTTP server.
+It supports setting url, method, headers, user\_agent, authentication
+and body. Only PUT and POST method is supported so far.
+
+Example config:
+
+```
+@version: 3.7
+@include "scl.conf"
+source      s_system { system(); internal(); };
+destination http_des {
+    http(
+        url("http://127.0.0.1:8000")
+        method("PUT")
+        user_agent("syslog-ng User Agent")
+        user("user")
+        password("password")
+        headers("HEADER1: header1", "HEADER2: header2")
+        body("BODY")
```

Comment:
Maybe show a templated body as example – e.g. `body("$MESSAGE")` – , to make it look more useful?